app/registries/foreman/access_control.rb

```diff
@@ -55,8 +55,8 @@ def remove_permission(permission)
 
       # Returns the actions that are allowed by the permission of given name
       def allowed_actions(permission_name)
-        perm = permission(permission_name)
-        perm ? perm.actions : []
+        perm = permissions&.filter { |p| p.name == permission_name }
```

Comment:
`filter` is only ruby > 2.6, that's why it fails on 2.5 thus the test failures.
```suggestion
        perm = permissions.select { |p| p.name == permission_name }
```